### DISCUSSION TITLE
建议：文件管理器，设置中，添加对 信息栏  的控制选项

### DISCUSSION THREAD

**Discussion by linuxlionking** [linuxdeepin/developer-center#3923] [Features Request & Ideas | 特性请求 & 头脑风暴]
如题，如果能控制启用 信息栏，会方便对文件进行预览，尤其是图片

当通过文管上传图片时，能快速确认图片是否选择正确

**Comment by BLumia** [comment]
我没看懂问题描述，是侧栏预览？现在的文件打开对话框是有侧栏预览按钮的呀。

具体什么问题，截个图看看？

**Reply by linuxlionking** [reply]
![截图_选择区域_20230322185540](https://user-images.githubusercontent.com/7598566/226882673-c4f76f74-6a16-409f-a501-0f53ef326fb6.png)
希望 能控制  信息栏 是否启用
现在需要手动启用
不知我是否描述清楚了

**Reply by BLumia** [reply]
cc @meiyixiang

**Comment by meiyixiang** [comment]
我不确定我理解的是否正确？你的意思应该是现在这个语言信息页面需要点击后才能看到，希望可以自定义是一直显示还是点击后显示？是这个意思吗？

**Reply by linuxlionking** [reply]
是的

**Reply by linuxlionking** [reply]
个人感觉，这样会增加易用性，具体是否可行，还请评估

**Reply by BLumia** [reply]
如果直接记住上次的状态是不是会好一些，这样不需要多一个设置项，并且符合用户习惯？

**Reply by linuxlionking** [reply]
这样实现更好，最终效果会更好

**Comment by meiyixiang** [comment]
如果理解不正确你可以更细致的描述问题。

**Comment by meiyixiang** [comment]
ok ,这个需求我已经记录反馈给文管项目组，但这个需求优先级不会很高做的化也会比较久。不过谢谢你给我们提出的需求建议。

**Reply by linuxlionking** [reply]
客气了。不妨事，本身就是个小的易用性改进，不要影响开发进度。

**Comment by meiyixiang** [comment]
问题诉求反馈，文管团队会综合考虑
